src/backend/modules/twitter/layout/css/twitter.css

```diff
@@ -2,4 +2,5 @@
     Settings
 */
 #twitter.twitterSettings #accounts { width: 320px; }
-#twitter.twitterSettings #hashtags { width: 320px; }
\ No newline at end of file
+
+#twitter.twitterSettings #hashtags { width: 320px; }
```

Comment:
kzie niet echt in waarom je dit hebt aangepast, maar bon, het zal wel niets kapot doen ;)